src/Tribe/Data_API.php

```diff
@@ -387,10 +391,16 @@ protected function get_attendees( $post_id, $context, $services = false ) {
 		if ( ! isset( $services['class'] ) && is_numeric( $post_id ) ) {
 			return Tribe__Tickets__Tickets::get_event_attendees( $post_id );
 		} elseif ( ! isset( $services['class'] ) && ! is_numeric( $post_id ) ) {
-			return array();
+			return [];
+		}
+
+		if( ! empty( $services['class'] ) ) {
```

Comment:
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after IF keyword; 0 found
* :no_entry_sign: There must be a space between if and the first parenthesis '('

<sub>hash: f9422216d259ed60a2d5ec92ffe3ecc4</sub>